utils/hare-reportbug

```diff
@@ -124,17 +120,15 @@ cd "$dest_dir/hare/$HOSTNAME"
 exec 5>&2
 exec 2> >(tee _reportbug.stderr >&2)
 
-if $use_systemd; then
-    systemd_journal
+systemd_journal
```

Comment:
have we tested this in container env?

Reply:
@mssawant Yes, we have tested it in container env.
Please check description.